docs/service-catalog/docs/020-architecture-service-catalog.md

```diff
@@ -2,16 +2,14 @@
 title: Architecture
 ---
 
-The diagram and steps describe the basic Service Catalog workflow:
+The diagram and steps describe the Service Catalog workflow and the roles of specific cluster and Environment-wide resources in this process:
 
-![Service Catalog flow](assets/service-catalog-flow.png)
+![Service Catalog flow](assets/service-catalog-flow.svg)
```

Comment:
Service Catalog frame should be more visible, more exposed and cover only service classes, brokers also should be excluded